src/components/AboutUs/StaffCard.js

```diff
@@ -35,57 +33,47 @@ const Description = styled(BodyPrimary)`
   padding-bottom: ${props => remcalc(30)};
 `
 
-const getSocialLink = ({ network, url }) => {
+const SocialLink = ({ name, url, image }) => {
   const size = `${IMAGE_SIZE}px`
 
-  switch (network) {
-    case 'twitter':
-      return (
-        <TappableAnchor href={url}>
-          <Image
-            image={{ file: { url: twitter } }}
-            alt="twitter profile link"
-            width={size}
-            height={size}
-          />
-        </TappableAnchor>
-      )
-    case 'linkedin':
-      return (
-        <TappableAnchor href={url}>
-          <Image
-            image={{ file: { url: linkedin } }}
-            alt="linkedin  profile link"
-            width={size}
-            height={size}
-          />
-        </TappableAnchor>
-      )
-    default:
-      return null
-  }
+  return (
+    <TappableAnchor href={url}>
+      <Image image={image} alt={name} width={size} height={size} />
+    </TappableAnchor>
+  )
 }
 
-const StaffCard = ({ name, image, title, description, socialLinks = [] }) => {
-  const links = (socialLinks || []).map(getSocialLink).filter(e => e)
-  const Links =
-    links.length > 0 ? (
-      <LinksContainer>{links.map((Link, idx) => Link)}</LinksContainer>
-    ) : null
+const SocialLinks = ({ data }) => {
+  return (data || []).length > 0 ? (
```

Comment:
I think a bit the other way around:
- making this check on the subcomponent makes the `StaffCard` render method simpler to read

If we were using this `data` on other subcomponent, then I'd agree that this could be done at the parent level, but for this case I think it hides some unnecessary complexity at the parent level...